snapcraft/parts/parts.py

```diff
@@ -185,11 +185,10 @@ def run(
                             project_vars=action.project_vars,
                             properties=action.properties,
                         )
-                    message = _action_message(action)
-                    emit.progress(f"Executing parts lifecycle: {message}")
-                    with emit.open_stream("Executing action") as stream:
+                    message = _get_parts_action_message(action)
+                    emit.progress(message)
+                    with emit.open_stream(message) as stream:
```

Comment:
Suggestion: bump craft-cli to 2.4.0 and then remove this parameter

Reply:
done